packages/Webkul/Shop/src/Resources/views/emails/sales/new-invoice.blade.php

```diff
@@ -156,12 +156,14 @@
                 </div>
             @endif
 
-            <div>
-                <span>{{ __('shop::app.mail.order.tax') }}</span>
-                <span style="float: right;">
-                    {{ core()->formatPrice($invoice->tax_amount, $invoice->order_currency_code) }}
+            @foreach (Webkul\Tax\Helpers\Tax::getTaxRatesWithAmount($refund, false) as $taxRate => $taxAmount)
```

Comment:
@AnnikaWolff 
In the old code is $invoice used. In our new code snipped we use $refund. Is this maybe a merge issue?

Reply:
Fixed - $invoice is correct.